Swiftilities.podspec

```diff
@@ -80,14 +80,23 @@ Pod::Spec.new do |s|
     ss.source_files = "Pod/Classes/FormattedTextField/*.swift"
     ss.frameworks   = ["UIKit"]
   end
-  
+
   # LicenseFormatter
 
   s.subspec "LicenseFormatter" do |ss|
     ss.source_files = "Pod/Classes/LicenseFormatter/*.swift"
     ss.frameworks   = "Foundation"
   end
 
+  # Acknowledgements
+
+  s.subspec "Acknowledgements" do |ss|
+    ss.dependency 'Swiftilities/LicenseFormatter'
+    ss.dependency 'Swiftilities/Deselection'
```

Comment:
I don't think it should, the use of `UIColor(hex:alpha:)` you're seeing is in the example unless I accidentally used it someone else too